.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
Please don't include the `.DS_Store` file in the repo

Reply:
I removed it and added .DS_Store files to the main gitignore, is that alright?